handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
```

Comment:
How is a trailing comma beneficial?

I have removed "Pieces" from the names, and the trailing brace is on a line following "Electrum"

Since I'm getting into enums here, would it be worth it to make one for Currency? 
```javascript
Currency = {
    Copper : "Copper",    // alternatively, Copper : 0
    Silver : "Silver",    // alternatively, Silver : 1
    // etc. I think using a string is nicer than a number, here, but let me know what you think
};